server/channels/app/webhook_test.go

```diff
@@ -867,8 +868,34 @@ func TestDoOutgoingWebhookRequest(t *testing.T) {
 		}))
 		defer server.Close()
 
-		resp, err := th.App.doOutgoingWebhookRequest(server.URL, strings.NewReader(""), "application/json")
+		resp, err := th.App.doOutgoingWebhookRequest(server.URL, strings.NewReader(""), "application/json", nil)
 		require.NoError(t, err)
 		require.Nil(t, resp)
 	})
+
+	t.Run("with auth token", func(t *testing.T) {
+		th.App.UpdateConfig(func(cfg *model.Config) {
+			cfg.FeatureFlags.OutgoingOAuthConnections = true
+		})
+		th.App.Srv().SetLicense(model.NewTestLicense("outgoing_oauth_connections"))
+
+		t.Cleanup(func() {
+			th.App.UpdateConfig(func(cfg *model.Config) {
+				cfg.FeatureFlags.OutgoingOAuthConnections = false
```

Comment:
I'm thinking we should set this back to whichever value it was set to beforehand. May not matter though